Formula/f3d.rb

```diff
@@ -0,0 +1,49 @@
+class F3d < Formula
+  desc "Fast and minimalist 3D viewer"
+  homepage "https://kitware.github.io/F3D/"
+  url "https://gitlab.kitware.com/f3d/f3d/-/archive/v1.0.1/f3d-v1.0.1.tar.gz"
+  sha256 "fb362dba3ccf49db9e8841d8a5310f37399bfea8866b6e040ce85670d54b97f7"
+  license "BSD-3-Clause"
+
+  depends_on "cmake" => :build
+  depends_on "vtk"
+
+  def install
+    # Is fixed upstream, remove this line for v1.1
+    inreplace "src/F3DOptions.cxx", "/etc/f3d/", "#{etc}/f3d/"
+
+    args = std_cmake_args + %W[
+      -DMACOSX_BUILD_BUNDLE:BOOL=OFF
+      -DBUILD_SHARED_LIBS:BOOL=ON
+      -DBUILD_TESTING:BOOL=OFF
+      -DF3D_INSTALL_DEFAULT_CONFIGURATION_FILE:BOOL=ON
+      -DCMAKE_INSTALL_NAME_DIR:STRING=#{lib}
+      -DCMAKE_INSTALL_RPATH:STRING=#{lib}
+    ]
+
+    mkdir "build" do
+      system "cmake", "..", *args
+      system "make"
+      system "make", "install"
+    end
+
+    mkdir_p etc/"f3d"
+    mv etc/"f3d/config.json", etc/"f3d/config.json.old" if File.exist?(etc/"f3d/config.json")
```

Comment:
This also won't work because it'll only ever run on CI.

Reply:
You mean that file system calls are not included when installing the binary package?

Reply:
They are only ran on CI when creating the binary. You need a `post_install` block to achieve this. But if it's only system config it should be always overwritten instead.

Reply:
Thanks, there was an issue upstream that I have fixed using `inreplace`. There is no need for file copies now.